index.html

```diff
@@ -138,12 +138,27 @@
   <button class="btn btn-medium font-bold"
   socialshare
   socialshare-provider="tumblr"
+  socialshare-type="link"
```

Comment:
I'm being explicit here, but you can leave this out as it will default to "link" anyway (for backwards compatibility)